NEWS.md

```diff
@@ -25,6 +25,18 @@ Thanks to @joseph-palmer, @RichardMN, and @kathsherratt for major contributions
 * `get_interventions_data()` is deprecated. These data no longer update as of December 2020. Check for alternatives at https://supertracker.spi.ox.ac.uk/policy-trackers/
 * `get_linelist` is deprecated. Linelist stopped updating June 2020. Up to date linelist data are now behind a login: access at https://global.health/. We are working on a solution for accessing with `covidregionaldata`.
 
+## Data changes since 0.8.3
+
+* Colombia now has capitalized region names.
+* ISO 3166-2 codes for CU-09, CU-08, CU-14 and CU-11 have been dropped for Cuba.
+* Germany level 2 region codes have been removed (previously was all NAs).
+* India uses NA for unknown region codes, a change from IN-UN previously.
+* Italy column region is now regioni.
+* Mexico codes 'inegi_code' has been changed to 'inegi'.
+* UK Level 1 'ons_region_code' is now 'iso_3166_2'.
```

Comment:
Are ONS region codes really iso_3166_2, or is this something which should be changed in the code?

Reply:
I must be honest I don't really know much about region codes and doing a little google around found impenetrable ONS documents where I couldn't find a deffinition for the codes. The closest I came is [this xls file](https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/populationandmigration/migrationwithintheuk/adhocs/005632internalmigrationforlocalauthoritiesinenglandandwalesbysingleyearofageandsexincludingflowstoandfromscotlandandnorthernirelandyearsendingjune2002tojune2010/laregionlookup376las.xls) which has the sames codes as us but called 'region_code' and 'local_authority_code' (la_code), perhapse this could work?

Reply:
LGTM - I think this is recognizable and doesn't present the codes as something which they may not be.